extensions.json

```diff
@@ -1096,6 +1096,11 @@
       "version": "0.10.1",
       "checkout": "v0.10.1"
     },
+    {
+      "id": "tomblind.local-lua-debugger-vscode",
+      "repository": "https://github.com/tomblind/local-lua-debugger-vscode",
+      "version": "0.2.0"
```

Comment:
```suggestion
      "version": "0.2.0",
      "checkout": "release-0.2.0",
      "prepublish": "npm run build"
```

Reply:
Thanks, I'll get it fixed. Pardon my ignorance, but how exactly would I know if for a given package this prepublish step is required? And would not the npm run publish also be needed? I just would like to have this clear on my mind so that I won't repeat mistakes.